pkg/api/mocks.go

```diff
@@ -53,7 +53,11 @@ func CreateMockContainerService(containerServiceName, orchestratorVersion string
 
 	cs.Properties.OrchestratorProfile = &OrchestratorProfile{}
 	cs.Properties.OrchestratorProfile.OrchestratorType = Kubernetes
-	cs.Properties.OrchestratorProfile.OrchestratorVersion = orchestratorVersion
+	if orchestratorVersion == "" {
```

Comment:
This addition of giving significance to `""` means that we can invoke this mock func without passing an explicit version, and interpret that as "just make me a mock cluster config with the current default version". This way as the set of supported versions evolves over time we don't have to keep manually updating a bunch of UT.